python/py-sacremoses/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        alvations sacremoses 99102ce
+version             0.0.35
+revision            0
+name                py-sacremoses
+
+license             GPL-2.1
+maintainers         nomaintainer
+platforms           darwin
+
+description         Python port of Moses tokenizer, truecaser and normalizer
+
+long_description    ${description}
+
+homepage            https://github.com/alvations/sacremoses
+
+python.versions     27 37 38
```

Comment:
I'm not sure new ports should be supporting 2.7 for anything; it's less than two months until it goes out of support permanently. @reneeotten will have an opinion on that I suspect.

Reply:
Thanks done.